internal/project/core.go

```diff
@@ -12,9 +12,11 @@ import (
 // ICore is an interface over project core
 type ICore interface {
 	CreateProject(ctx context.Context, m *Model) error
+	Get(ctx context.Context, key string) (*Model, error)
 	Exists(ctx context.Context, key string) (bool, error)
 	ExistsWithID(ctx context.Context, id string) (bool, error)
 	DeleteProject(ctx context.Context, m *Model) error
+	UpdateProject(ctx context.Context, m *Model) error
```

Comment:
what can be updated in a project ?

Reply:
add unit test for this please

Reply:
Had added this for the older approach where creds were being stored as part of the project model itself. Not needed now. Will remove.